src/core/Expander/Expander/Expander.baseStyles.tsx

```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { element, font } from '../../theme/reset';
+
+import { ExpanderProps } from './Expander';
+
+/* stylelint-disable no-descending-specificity, no-duplicate-selectors */
```

Comment:
I think this ignore comment is unneeded. Couldn't produce any linter issue when removed the line.

Reply:
Will remove. It was needed before breaking down the styles.

Reply:
Removed

Reply:
It seems that it got removed from other baseStyles, but were still left in this file.

Reply:
So it seems. Probably did one too many undos while testing out something. Will fix shortly.

Reply:
Fixed.